components/shared/Home/ContributionsTracker.tsx

```diff
@@ -1,133 +1,86 @@
+  function getClassNameValue(value: any) {
+    if (!value || value.count === 0) {
+      return `color-myCal-0`;
+    } else if (value.count < 2) {
+      return `color-myCal-1`;
+    } else if (value.count < 3) {
+      return `color-myCal-2`;
+    } else if (value.count < 4) {
+      return `color-myCal-3`;
+    } else if (value.count < 5) {
+      return `color-myCal-4`;
+    } else {
+      return `color-myCal-5`;
     }
-
-    return days;
-  };
-
-  const days = generateCalendar();
-  const weeks: { date: string; count: number }[][] = [];
-  for (let i = 0; i < days.length; i += 7) {
-    weeks.push(days.slice(i, i + 7));
   }
 
-  const renderWeek = (week: { date: string; count: number }[]) => {
-    return (
-      <div className="flex gap-2">
-        {week.map((day, index) => {
-          return (
-            <div
-              key={index}
-              className={`size-2  rounded-md text-subtitle ${
-                day.count > 0 ? "bg-myGreen-500" : "bg-myWhite-500"
-              }`}
-            >
-              {day.date}
-            </div>
-          );
-        })}
-      </div>
-    );
-  };
-
-  //get current month
-  const month = new Date().toLocaleString("default", { month: "long" });
-  //get months from current month to 12 months ago
-  const months = [];
-  for (let i = 13; i > 1; i--) {
-    months.push(
-      new Date(new Date().setMonth(new Date().getMonth() - i)).toLocaleString(
-        "default",
-        { month: "short" }
-      )
-    );
-  }
+  const rectClasses = [
+    "color-myCal-1",
+    "color-myCal-2",
+    "color-myCal-3",
+    "color-myCal-4",
+    "color-myCal-5",
+  ];
 
-  //get days of the week
-  const daysOfWeek = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"];
+  const isMedDevice = useMediaQuery("only screen and (max-width : 768px)");
 
   return (
-    <div className=" flex flex-col border border-myWhite-300 bg-myBlack-900 p-4">
-      <div className="mt-4 flex justify-around gap-2">
-        {months.map((month, index) => {
-          return (
-            <div key={index} className="text-p4Med text-myWhite-300">
-              {month}
-            </div>
-          );
+    <div className=" rounded-sm border border-myWhite-500 bg-myBlack-900 p-4">
+      <CalendarHeatmap
+        startDate={lastYear}
+        endDate={new Date()}
+        values={testContributions.contributions.map((contribution) => {
+          return {
+            date: contribution.date,
+            count: contribution.count,
+          };
         })}
-      </div>
-      <div className="flex">
-        <div className="flex flex-col gap-2">
-          {daysOfWeek.map((day, index) => {
-            return (
-              <div key={index} className="text-p4Med text-myWhite-300">
-                {day}
-              </div>
-            );
-          })}
+        showWeekdayLabels={true}
+        gutterSize={4}
+        weekdayLabels={DAY_LABELS}
+        classForValue={(value) => getClassNameValue(value)}
```

Comment:
```suggestion
        classForValue={getClassNameValue}
```

Minor suggestion- we don't need to make an anonymous function here.